src/geo/transform.js

```diff
@@ -149,6 +149,7 @@ class Transform {
     _nearZ: number;
     _farZ: number;
     _mercatorScaleRatio: number;
+    _globeZoomTransition: boolean;
```

Comment:
```suggestion
    _globeZoomTransition: boolean;
```
I'm not sure about staring this value in `transform`, it seems simpler just to pass it through `map._updateProjection`. Or do we anticipate a need to use this value elsewhere?

Reply:
Ah was trying to avoid the confusing passing of arguments. seeing your comment about an object is better than boolean (will change)